uwblueprint/community-fridge-kw
# feat: frontend CheckInAPIClient

## Brief description. What is this change? 

### [Create Check-in API Clients](https://www.notion.so/uwblueprintexecs/Create-Check-in-API-Clients-567bba9e49c6488684b96b8c66df6372)



## Implementation description. How did you make this change?
* created frontend `CheckInAPIClient.ts` for all checkin services (get, delete, post, put) in backend and `CheckInTypes.ts`
* did sanity check test with delete/get methods with button to verify calls were being made
* NOTE: the create and update routes HAVEN'T been merged into staging yet, so haven't tested these services yet



## Steps to test
1. Create new CheckIn entries in pgadmin
2. Create a button that calls function that make calls using the new CheckInAPIClient (make sure you import CheckInAPIClient in the file)
eg `CheckInAPIClient.deleteCheckIn(checkInId)` with checkInId corresponding with a checkin entity in pgadmin table
3. Click the button and see if the check in is deleted
4. Make other calls to other route calls (i.e getting all checkins (and printing response to console), etc)


## Checklist
